lifecycle_test.go

```diff
@@ -363,11 +377,12 @@ func uploadBuildWithDeps(t *testing.T, buildName, buildNumber string) {
 	runRt(t, "build-publish", buildName, buildNumber)
 }
 
-func initLifecycleTest(t *testing.T) (cleanCallback func()) {
+func initLifecycleTest(t *testing.T, minVersion string) (cleanCallback func()) {
```

Comment:
There's probably no need to pass `minVersion` as an argument for this function, because the `signingKeyOptionalArtifactoryMinVersion` const should already be accessible for it.

Reply:
The initLifecycleTest function is used across multiple tests. However, for the signingKey test, I need to ensure it includes the minimum RT version required to support this API, as this parameter is necessary.

Reply:
@oshratZairi,
The `initLifecycleTest` function is used only from within the `lifecycle.go` file.
This function can access the `signingKeyOptionalArtifactoryMinVersion` const on its own, and therefore there's no need for the function to recieve it as an argument.